aws-cpp-sdk-core/CMakeLists.txt

```diff
@@ -402,6 +397,13 @@ else()
     endif()
 endif()
 
+configure_file("${CMAKE_SOURCE_DIR}/toolchains/pkg-config.pc.in" "${PROJECT_NAME}.pc" @ONLY)
```

Comment:
Also, I've updated the generator to automatically replicate the changes for other CMakeFiles and stripped them from the generated files hopefully so you shouldn't have issues trying to merge it locally :)